crypto_track/blog.md

```diff
@@ -0,0 +1,574 @@
+# Crypto Track: Cryptocurrency tracking web application with instant email notifications
+
+## Background
```

Comment:
I'd recommend adding either screenshots/gifs or embedding your demo video at the top.

Reply:
Done. I have added a link to the demo video